scripts/caseMapping/CaseMapping.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE OverloadedStrings #-}
+module CaseMapping (
+  main
+) where
+
+import qualified Basement.String as BS (charMap)
+import           Data.Char (toUpper, toLower)
+import           Foundation
+import           Foundation.IO 
+import qualified Foundation.String as S
+
+import CaseFolding
+import SpecialCasing
+
+main = do
+  psc <- parseSC "SpecialCasing.txt"
+  pcf <- parseCF "CaseFolding.txt"
+  scs <- case psc of
+           Left err -> putStrLn (show err) >> undefined
+           Right sc -> return sc
+  cfs <- case pcf of
+           Left err -> putStrLn (show err) >> undefined
+           Right cf -> return cf
+  h <- openFile ("../../basement/Basement/String/CaseMapping.hs") WriteMode
```

Comment:
might be easier to have the haskell program write to stdout and redirect the stdout to the file when wanting to rewrite